### PR TITLE
SAK-44787 Update sakai_21_mysql_conversion.sql

### PR DESCRIPTION
**Pull request by csev**
Need some help with the Oracle bits @bjones86  - Thanks in advance

### CODE REVIEW

**Comment by bjones86** [comment]
Oracle equivalent:

```
ALTER TABLE lti_tools ADD pl_coursenav NUMBER(1,0) DEFAULT 0;
```

File: docs/conversion/sakai_21_mysql_conversion.sql [hunk 1]
```diff
@@ -19,3 +19,6 @@ ALTER TABLE rbc_criterion ADD COLUMN WEIGHT DOUBLE NULL DEFAULT 0;
 -- Existing records needed to be switched on right before the feature is used
 UPDATE lti_tools SET pl_lessonsselection = 1;
 -- END SAK-44637
+
+-- SAK-44787 - Add the field early - code that uses this will come later
+ALTER TABLE lti_tools ADD     pl_coursenav TINYINT DEFAULT '0'
```
Comment (bjones86) [inline docs/conversion/sakai_21_mysql_conversion.sql:24]: The default should be `0`, not `'0'` (number vs. string):

```suggestion
ALTER TABLE lti_tools ADD pl_coursenav TINYINT DEFAULT 0;
```

**Review by bjones86 — CHANGES_REQUESTED** [review]

**Comment by bjones86** [comment]
Please remember to update the commit message with the SAK # when merging this.

**Comment by ern** [comment]
Closing, as this is already incorporated

**Pull request closed without merging**